### DISCUSSION TITLE
Should we consider SwayOSD instead of Swaync based volume/brightness and key states?

### DISCUSSION THREAD

**Discussion by SherLock707** [JaKooLit/Hyprland-Dots#294] [Ideas]
[https://github.com/ErikReider/SwayOSD](https://github.com/ErikReider/SwayOSD)

Since muting notifications in Swaync takes away volume and other stuff indicator too..

**Comment by JaKooLit** [comment]
I dont know. I dont like the idea actually lmao.. its just added bug to introduce.. imho

do you have issues with swaync?

**Reply by SherLock707** [reply]
No issue as such...

Once I muted notification since I was watching some, then tried to increase value, I wasn't getting the volume and brightness indicators since they were muted as well since OSD and notification is same here..

**Comment by JaKooLit** [comment]
ah.. got what you mean.. swaync does not pick up volume controls which is not linked to the volume script

**Reply by SherLock707** [reply]
Only if there was a way to exclude volume and brightness notification from dnd mode.

**Reply by JaKooLit** [reply]
should we approch dev?

**Reply by SherLock707** [reply]
> should we approch dev?

I'll try to add something in evening today..

**Reply by SherLock707** [reply]
Apologies for delay, life caught up with me.

Used SwayOSD for few days, its cool but felt too limited in terms of user configuration and it does not respect any key remapping.
So decided to drop it...

Also I feel we should replace everything with AGS eventually! ✨

**Reply by SherLock707** [reply]
@JaKooLit , lets start with 'ags' tag in repo description part. 😂

**Reply by JaKooLit** [reply]
> Apologies for delay, life caught up with me.
> 
> Used SwayOSD for few days, its cool but felt too limited in terms of user configuration and it does not respect any key remapping. So decided to drop it...
> 
> Also I feel we should replace everything with AGS eventually! ✨

not an issue bro... we all need to do our own things to do...

let me get back to my computer later today and create a separate branch for ags

using phone is meh... hard and things can go sideways
